scripts/setup.sh

```diff
@@ -11,6 +11,7 @@ python_version="python39"
 pip_version="pip3.9"
 llm_master_dir="/home/opc/vllm-master"
 parallel_gpu_count=4
+vllm_log_file="vllm_log_file.log"
```

Comment:
Please don't remove the log export from a script file .
We move the log file to bas.sh or bash_openai.sh because clients want to change the log file later to add it in OS log rotation. LLM generates lots of logs 
In that to take effect it must be part of the service script (bash.sh or bash_openai.sh) and the line 64 should be >>${vllm_log_file} 2>&1